nms/app/packages/magmalte/server/app.js

```diff
@@ -104,7 +104,13 @@ app.use(configureAccess({loginUrl: '/user/login'}));
 // Grafana uses its own CSRF, so we don't need to handle it on our side.
 // Grafana can access all metrics of an org, so it must be restricted
 // to superusers
+const fs = require('fs');
 app.use('/grafana', access(SUPERUSER), require('../grafana/routes.js').default);
+app.get('/data/LteMetrics', (req, res) => {
+  fs.readFile(path.join(__dirname, '..', 'data/LteMetrics.json'), (_, json) => {
```

Comment:
can we load this into the router beforehand so we don't have to make a fs request every time?